project/en-US/jts/trailmap.xml

```diff
@@ -1504,7 +1503,7 @@ TransactionManager.commit();
                 when the transaction terminates, e.g., are driven through the two-phase commit
                 protocol.
                 <para>
-                    The JTA is much more closely integrated with the XA concept of resources
+                    Jakarta Transactions is much more closely integrated with the XA concept of resources
```

Comment:
Jakarta Transactions **are** much more closely integrated with the XA concept of resources